src/components/CareerFramework/Discipline.js

```diff
@@ -0,0 +1,49 @@
+import React, { Fragment } from 'react'
+import generate from 'shortid'
+import styled from 'styled-components'
+
+import { Grid } from '../grid'
+import GreyBackground from '../Common/GreyBackground'
+import BlueBackground from '../Common/BlueBackground'
+
+import Group from './Group'
+import Join from './Join'
+
+const StyledGrid = styled(Grid)`
+  padding-bottom: ${({ theme, isLastGroupWithoutJoins }) =>
+    isLastGroupWithoutJoins ? theme.space[7] : null};
+`
+
+const Discipline = ({ groups, joins = [] }) => (
+  <>
```

Comment:
I have a question: What is the advantage of using something like `<>`as opposed to `<Fragment>` here?